package.json

```diff
@@ -89,7 +89,6 @@
   "jsonwebtoken": "^8.5.1",
   "multer": "^1.4.2",
   "pg": "^8.3.0",
-  "reflect-metadata": "^0.1.13",
```

Comment:
I think this might have been accidentally deleted, could you add it back? It's required by TypeORM